plugin/trino-vertica/src/main/java/io/trino/plugin/vertica/VerticaClientModule.java

```diff
@@ -62,7 +64,14 @@ protected void setup(Binder binder)
     @ForBaseJdbc
     public static ConnectionFactory createConnectionFactory(BaseJdbcConfig config, CredentialProvider credentialProvider, OpenTelemetry openTelemetry)
     {
+        // In vertica all the underlying projections data stored according to en_US@collation=binary
+        // So to be safe with pushdowns and guarantee the same results for trino with and without pushdowns
+        // we should set session level collation to en_US@collation=binary,
+        // in this case setting up database level collation (for all incoming sessions) will not break pushdown functionlaity
+        Properties properties = new Properties();
+        properties.setProperty("ConnSettings", "SET LOCALE TO 'en_US@collation=binary'");
```

Comment:
Does this line override or append `ConnSettings` property user set via connection string? I'm asking this question because `ConnSettings` can be used for other purposes, e.g. search path.

Reply:
Great catch, it's override it, I need to move this logic to custom VericaConnectionFactory, where I can set this collation by explicit call